cypress/integration/createTask.js

```diff
@@ -0,0 +1,22 @@
+import TaskPage from "../objects/taskPage";
+
+const taskPage = new TaskPage();
+let taskName = Math.random().toString(36);
+
+describe('Sourcebooks create task', function () {
+    it('Should be able to create new task as admin', function () {
+        cy.loginAs('Admin');
```

Comment:
Suggestion to move this outside current function as a nature of the test files is to have more than one test case (function)